CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 - Multiple modulo functions for the `Eval*` nodes: `mod_e` (euclidean), `mod_f`
   (floored) and `mod_t` (truncated)
 - Render to 3D textures support
+- Image load/store for 3D textures support
```

Comment:
Ditto trailing "support"; maybe "Support for 3D textures image load/store"

Reply:
Done.